### PR TITLE
[TypeInfo] add BC breaking TypeInfo changes to upgrade file

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Review by wouterj — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

File: UPGRADE-7.2.md [hunk 1]
```diff
@@ -112,6 +112,13 @@ TwigBridge
 
  * Deprecate passing a tag to the constructor of `FormThemeNode`
 
+TypeInfo
+--------
+
+ * Rename `Type::isA()` to `Type::isIdentifiedBy()` and `Type::is()` to `Type::isSatisfiedBy()`
+ * Remove `Type::getBaseType()`, `Type::asNonNullable()` and `Type::__call()` methods
```
Comment (wouterj) [inline UPGRADE-7.2.md]: Can we maybe add relevant replacements for `getBaseType()` and `asNonNullable()`?

I personally couldn't find it quickly when checking out the pull request, which means it's probably even harder for users to find a way to fix their code.

Comment (xabbuh) [reply UPGRADE-7.2.md]: ping @mtarld

Comment (mtarld) [reply UPGRADE-7.2.md]: Yes, you're right, we could write something like that instead?
```
* Remove `Type::getBaseType()`, use `WrappingTypeInterface::getWrappedType()` instead
* Remove `Type::asNonNullable()`, use `NullableType::getWrappedType()` instead
```

Comment (fabpot) [reply UPGRADE-7.2.md]: That works for me.

Comment (xabbuh) [reply UPGRADE-7.2.md]: 👍 updated

**Review by mtarld — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**